Dao/ContainerReleaseDao.php

```diff
@@ -168,6 +168,11 @@ public function deleteNoLongerExistingEnvironmentReleases($availableEnvironments
         return $query->rowCount();
     }
 
+    protected function isNameAlreadyUsed(int $idSite, string $name, ?int $idContainerVersion = null): bool
+    {
+        return true;
```

Comment:
@snake14 Did we forgot to add some logic to check the name here ?

Reply:
@AltamashShaikh No. Releases don't have a name, so this dao doesn't use the method, but has to implement it since it's abstract. I added a comment to explain why the return value is hard coded.